enhancements/rhcos/split-rhcos-into-layers.md

```diff
@@ -0,0 +1,458 @@
+---
+title: split-rhcos-into-layers
+authors:
+  - "@jlebon"
+  - "@cverna"
+  - "@travier"
+reviewers:
+  - "@patrickdillon, for installer impact"
+  - "@rphillips, for node impact"
+  - "@joepvd, for ART impact"
+  - "@sinnykumari, for MCO impact"
+  - "@LorbusChris, for OKD impact"
+  - "@zaneb, for agent installer impact"
+  - "@sdodson, for overall architecture"
+  - "@cgwalters, for overall architecture"
```

Comment:
@fabiendupont @bthurber @ybettan please review in a context of kmm and dtk

Reply:
Since KMM and DTK are day 2 in the case of OCP, I don't see any real impact or change here.  We aren't managing anything as day 0.